Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -2877,12 +2881,18 @@ static bool isContextMenuEvent(const WebMouseEvent& event)
     };
 }
 
-Vector<PDFContextMenuItem> UnifiedPDFPlugin::navigationContextMenuItems() const
+Vector<PDFContextMenuItem> UnifiedPDFPlugin::navigationContextMenuItems(PDFDocumentLayout::PageIndex contextMenuEventPage) const
```

Comment:
You could rename this `navigationContextMenuItemsForPageAtIndex()` and then the arg could just be `pageIndex`